src/modules/CMakeLists.txt

```diff
@@ -102,8 +102,11 @@ if( "${BACKEND}" STREQUAL "HIP")
 
     # Set compiler flags
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${HIP_HIPCC_FLAGS}")
-    set_source_files_properties(rppt_tensor_audio_augmentations.cpp PROPERTIES COMPILE_FLAGS -mno-fma)
-    set_source_files_properties(rppt_tensor_statistical_operations.cpp PROPERTIES COMPILE_FLAGS -mno-fma) # no-fma flag added to get the exact output match with golden outputs
+    set_source_files_properties(tensor/rppt_tensor_audio_augmentations.cpp tensor/rppt_tensor_statistical_operations.cpp PROPERTIES COMPILE_FLAGS -mno-fma)
+    set_source_files_properties(tensor/cpu/kernel/non_silent_region_detection.cpp tensor/cpu/kernel/to_decibels.cpp tensor/cpu/kernel/pre_emphasis_filter.cpp tensor/cpu/kernel/down_mixing.cpp tensor/cpu/kernel/spectrogram.cpp tensor/cpu/kernel/slice.cpp tensor/cpu/kernel/resample.cpp tensor/cpu/kernel/mel_filter_bank.cpp PROPERTIES COMPILE_FLAGS -mno-fma)
```

Comment:
Check adding in Cmake custom function and merging all set_source_files_properties to one call inside the custom function